src/ClientHandle.h

```diff
@@ -408,6 +410,13 @@ class cClientHandle  // tolua_export
 
 	cPlayer * m_Player;
 
+	/** This is an optimization which saves you an iteration of m_SentChunks if you just want to know
+	whether or not the player is standing at a sent chunk.
+	If this is equal to the coordinates of the chunk the player is currrently standing at, then this must be a sent chunk
+	and a member of m_SentChunks.
+	Otherwise, this contains an arbitrary value which should not be used. */
+	cChunkCoords m_CachedSentChunk;
```

Comment:
Don't bother. It's not worth it for a new variable, and besides, bugs will manifest when teleporting to ChunkX + 500.